packages/main/src/themes/Toolbar.css

```diff
@@ -0,0 +1,122 @@
+:host(:not([hidden])) {
+	display: inline-block;
+	width: 100%;
+	height: 2.75rem;
+	background: var(--ui5-toolbar-Background);
+	border-bottom: 0.0625rem solid var(--sapGroup_ContentBorderColor);
+	flex-shrink: 1;
+	color: var(--sapButton_TextColor);
+}
+
+:host(:not([hidden])[data-ui5-compact-size]) {
+	height: 2rem;
+}
+
+:host([disabled]) {
+	opacity: .5;
+	pointer-events: none;
+}
+
+:host([design="Solid"]) {
+	background: var(--sapBaseColor);
+}
+
+:host([design="Transparent"]) {
+	background: var(--sapToolbar_Background);
+}
+
+:host([styling="Clear"]) {
+	border-bottom: 0;
+}
+
+.ui5-tb-root {
+	/*min-width: 2.5rem;*/
+	width: 100%;
+	padding: 0 0.5rem;
+	justify-content: flex-end;
+	align-items: center;
+	overflow: hidden;
+	pointer-events: none;
```

Comment:
Although we discussed this, on a second thought it may be unnecessary, as it works correctly when I test without it now. The toolbar container does not receive the focus when it gets clicked even withoutt the pointer events being set to none. Pertpahs, there was a caching issue when we looked into it. Let me know if it still reprodices on your browser, we can look further into the root of the problem.